### ISSUE TITLE
[BITV] 9.1.1.1a/3.1 - Modal Dialog "Choose Target Folder" - The Home button lacks an alternative text. An empty a-element without a href attribute was used, placed within a div element with tabindex="0". (2)

### ISSUE THREAD

**Issue by Pytal** [nextcloud/server#36714]
- [x] https://github.com/nextcloud/server/pull/36741
- [x] https://github.com/nextcloud/server/pull/36858

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-cd239865a5-v3-n1

![image](https://user-images.githubusercontent.com/24800714/218899862-17a5c22a-60eb-4513-8ef1-2b19adb2d294.png)

</details>

-  [x] 9.1.1.1a/10.1 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-cd239865a5-v10-n1
